spec/takeaway_spec.rb

```diff
@@ -0,0 +1,18 @@
+require './lib/takeaway'
+require 'twilio-ruby'
+
+describe Takeaway do
+  
+  let(:dummy_messages) { double :dummy_messages }
+  let(:client) do
+    double :client, messages: dummy_messages
+  end
+
+  let(:beijing_hanoi) {double :person, number: '441322721370', name: 'BeijingHanoi'}
+  let(:nat) {double :person, number: '447921262696', name: 'Nat'}
+
+  it 'sends a message' do
+  	expect(dummy_messages).to receive :create 
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.